新增爬取課表

LIAO-JIAN-PENG opened the pull request:
內有print做測試，使用後刪除

LIAO-JIAN-PENG committed:
新增爬取課表

內有print做測試，使用後刪除

LIAO-JIAN-PENG committed:
改回去爬選課系統(強調還未整理code)

這次終於解決POST問題，然後也還未整理，所以僅供參考，但是時間應該是做好了，明天有空再弄

JiaxuanTW commented:
這麼有效率，效果不差嘛~
我把格式也給你，不過真的不急，你有空再看看就好。

```
{ 
    { 
        'name':"CourseName",
        'credit':"0",
        'time':[ [2, [3, 4]], [4, [3, 4]] ],
        'location':"Location",
        'teacher':"TeacherName",
    }
}
```

LIAO-JIAN-PENG committed:
增加teacher and credit 

因為各分布在不同網頁，所以導致速度被拉慢

LIAO-JIAN-PENG committed:
函式化

   {
      "name":"服務學習培養(資訊工程學系A)",
      "credit":0,
      "time":[
         [
            1,
            [
               "X"
            ]
         ]
      ],
      "location":[
         ""
      ],
      "teacher":"賴祥慶"
   },
   {
      "name":"體育(三)-桌球A",
      "credit":0,
      "time":[
         [
            3,
            [
               "1",
               "2"
            ]
         ]
      ],
      "location":[
         ""
      ],
      "teacher":"馮潤賓"
   }
]
